core/src/components/radio-group/radio-group.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../themes/ionic.globals";
+
+// Radio Group
+// --------------------------------------------------
+
+ion-radio-group {
+  // Prevents additional pixels from being rendered on top
+  vertical-align: top;
+}
+
+.radio-group-wrapper {
+  display: inline;
+}
+
+// Radio Group: Top
+// --------------------------------------------------
+
+.radio-group-top {
+  line-height: 1.5;
+}
+
+/**
+ * Error text should only be shown when .ion-invalid is present
+ * on the radio group. Otherwise the helper text should be shown.
+ */
+.radio-group-top .error-text {
+  display: none;
+
+  color: ion-color(danger, base);
+}
+
+.radio-group-top .helper-text {
+  display: block;
+
+  color: $text-color-step-300;
+}
+
+.ion-touched.ion-invalid .radio-group-top .error-text {
+  display: block;
+}
+
+.ion-touched.ion-invalid .radio-group-top .helper-text {
+  display: none;
```

Comment:
I don't really know if this is worth mentioning, and it may be expected behavior, but this means helper text will be hidden in an invalid state even if there's no error text. That might be confusing, but it's probably a non-issue.

Reply:
Correct! I actually mentioned this the other day because this is how both the [input](https://github.com/ionic-team/ionic-framework/blob/8eaeb22e7a967100ffaadae8c8221e2e4888a3b6/core/src/components/input/input.scss#L346-L352) and [textarea](https://github.com/ionic-team/ionic-framework/blob/8eaeb22e7a967100ffaadae8c8221e2e4888a3b6/core/src/components/textarea/textarea.scss#L423-L429) components (the only two that are already released) currently behave. I thought this was a weird behavior, but I didn't change it since it aligns with the existing implementation of those components. Maybe we can revisit this in the future.

Reply:
My guess is that developers might want this behavior. I wouldn't be surprised that there are some who don't want the error text but need the classes for them to do their own validation.